idelib/dataset.py

```diff
@@ -2222,6 +2216,9 @@ def arrayMinMeanMax(self, startTime=None, endTime=None, padding=0,
             :return: A structured array of data block statistics (min, mean,
                 and max, respectively).
         """
+        # TODO: Remember what `padding` was for, and either implement or
+        #   remove it completely. Related to plotting; see `plots`.
+        # TODO: Use `iterator`? It may have been removed accidentally.
```

Comment:
Should these also be Jira tickets? I worry that comments are easier to lose

Reply:
> Should these also be Jira tickets? I worry that comments are easier to lose

These are pretty vague, and neither are critical. I'm not sure it's worth cluttering up Jira; there are already a bunch of old GitHub 'enhancement' issues from different repos, and they sort of drown out the more critical issues.